plugins/catalog/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@backstage/core": "^0.1.1-alpha.6",
     "@backstage/theme": "^0.1.1-alpha.6",
+    "@backstage/catalog-model": "^0.1.1-alpha.6",
```

Comment:
Why this change?

Reply:
The catalog plugin refers to a type from the catalog-model [here](https://github.com/spotify/backstage/blob/master/plugins/catalog/src/data/utils.ts#L17) and the dep is not present  in [package.json](https://github.com/spotify/backstage/blob/master/plugins/catalog/package.json) in master 

I ran `yarn run lint` locally on my branch and master and it errored out saying the dependency is not added to package.json. 

Something is up with the linting setup in GitHub workflows maybe?

Maybe @freben @Rugvip can take a look at it.

Reply:
Seems like a correct fix, but make a separate PR for it imo.